pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -663,13 +627,19 @@ class Build {
 
         context.node('worker') {
             context.stage("installer") {
-                // Ensure master context workspace is clean of any previous archives
-                context.sh "rm -rf workspace/target/* || true"
 
                 switch (buildConfig.TARGET_OS) {
-                    case "mac":     buildMacInstaller(versionData); break
-                    case "windows": buildWindowsInstaller(versionData); break
-                    default: break
+                    case "mac":
+                        context.sh 'rm -f workspace/target/*.pkg workspace/target/*.pkg.json workspace/target/*.pkg.sha256.txt';
```

Comment:
If https://github.com/adoptium/ci-jenkins-pipelines/pull/371#issuecomment-1202674831 do the cleanup

Why mac need extra cleanup?

Reply:
the function of `cleanWorkspaceBuildOutputAfter` was added by Morgan Davies on 2021.02.04 from 485a9775ec8a353da63abdd4e9a5cc2261139e93
the `"context.sh 'rm -f....."` for `case "mac"` was added by Adam Farley on 2021.01.07 9537297119aa13bc1a2a9e86ebf550b0428c93f6
it was the same case when linux as in the same switch-case without cleanup, till e18d9e9c59e6079b48b57701fcb40074775b727c remove it